plugin/ipam/main.go

```diff
@@ -89,7 +89,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 	// conf.ReuseIPWait seconds old in the registry to be
 	// considered for use.
 	free, err := aws.FindFreeIPsAtIndex(conf.IfaceIndex, true)
-	if err == nil && len(free) > 0 {
+	if err == nil || len(free) > 0 {
```

Comment:
why not && here?

Reply:
Same as the previous `FindFreeIPsAtIndex` instance